tests/vmi_configuration_test.go

```diff
@@ -423,7 +433,7 @@ var _ = Describe("[sig-compute]Configurations", func() {
 				Expect(domXml).To(ContainSubstring("queues='3'"))
 			})
 
-			It("[test_id:1665]should map cores to virtio net queues", func() {
+			DescribeTable("should map cores to virtio net queues", func(updateSpec func(vmi *v1.VirtualMachineInstance)) {
```

Comment:
Adding an e2e test for this issue seems an overkill to me.
If we will start covering all the potential scenarios, we will add tens of such tests.
Given the e2e are expensive, I think we should just depend on the unit tests covering this well enough.

Reply:
Removed the e2e test